### PR TITLE
New ContentType: application/schema+json.jsonld

### PR DESCRIPTION
**Pull request by lzehl**

### CODE REVIEW

**Comment by lzehl** [comment]
@apdavison OR @jagru20: could one of you review this content type addition?

**Comment by jagru20** [comment]
sorry, this slipped my attention! Looks good to me.

File: instances/data/contentTypes/schema+json.jsonld [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	},
+	"@id": "https://openminds.ebrains.eu/instances/contentTypes/application/schema+json",
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+	"fileExtension": [
+		".schema.json",
+    ".json",
+    ".jschema",
+    ".jsd",
+    ".jsonsd"
+	],
```
Comment (jagru20) [inline instances/data/contentTypes/schema+json.jsonld]: indentation could be corrected here. (cosmetic change though)

**Review by jagru20 — APPROVED** [review]

**Comment by lzehl** [comment]
@jagru20 feel free to merge then

File: instances/data/contentTypes/schema+json.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/data/contentTypes/schema+json.jsonld]: done

**Review by jagru20 — APPROVED** [review]

File: instances/data/contentTypes/schema+json.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/data/contentTypes/schema+json.jsonld]: btw: this seems to be an GitHub issue... the indent was correct when I created the file... And when I reopen the file the indent is messed up. any idea why this happens?

Comment (jagru20) [reply instances/data/contentTypes/schema+json.jsonld]: I think I noticed the same for some contenttypes I created. I am not sure, but the most plausible explanation to me is that the way, the indentation is done (whitespaces vs. tabs) could be related to this somehow. I'd expect whitespaces to work better, but it's just a guess though.

**Pull request merged**